### PR TITLE
Generate client omicron be46344 for 0.1.0-rc.39

### PR DESCRIPTION
**Pull request by karencfv**
Updates generated client.

Due to https://github.com/oxidecomputer/omicron/pull/1329#issuecomment-1175806211, the spec will me modified to use the old operation IDs

### CODE REVIEW

**Comment by karencfv** [comment]
I am unsure why this GH action is failing in the "commit changes if any" step:

```console
Your branch is up to date with 'origin/generate-client-omicron-be46344-for-0.1.0-rc.39'.
nothing to commit, working tree clean
Error: Process completed with exit code 1.
```

Even stranger, it seems this GH action runs randomly 😅 https://github.com/oxidecomputer/oxide.rs/actions/workflows/make-generate.yml

@augustuswm Might you have any idea as to why this is happening?

File: Makefile [hunk 1]
```diff
@@ -17,12 +17,14 @@ target/debug/generator: generator/src/*.rs generator/Cargo.toml
 	cargo build --bin generator
 
 clean-spec:
-	$(RM) $(SPEC)
```
Comment (augustuswm) [inline Makefile:20]: Is this an intended change? (Meant to link against the stray `e` that was added)

**Comment by augustuswm** [comment]
I haven't looked at this repo before, but this is my initial take:

The action looks set to run whenever a PR changes `spec.json` (or the workflow itself):
```
on:
  pull_request:
    paths:
      - spec.json
      - .github/workflows/make-generate.yml
```

The workflow is failing as it is expecting that a change to `spec.json` implies that running `make generate` will create changes. So when it runs `git commit ...` it is returning an exit code of 1 as there are no files to commit. Effectively the workflow wants to be the only one updating code in `oxide/src`.

That step of the workflow probably should be changed so that it can still succeed even if it does not generate any changes.

**Comment by karencfv** [comment]
Ahhhhhhhhhh, of course, gotcha. I brought back the spec.json file as we have a temporary workaround for operation IDs, so that must have triggered it. Thanks @augustuswm, much appreciated! 🙇‍♀️

File: Makefile (hunk 1 above)
Comment (karencfv) [reply Makefile:20]: yep, for some reason sed is creating a `spec.jsone` file 🤷‍♀️ . Decided to leave it as this is a temporary workaround until the progenitor code is up and running (we won't need sed anymore with it)

**Comment by karencfv** [comment]
Removing the action all together. Like @augustuswm mentioned, it only gets triggered when the spec.json file changes. This means that it is a remnant from the old language client daisy chain that was already removed. The SDK will not be using the copy pasted spec.json file anymore, as it pulls directly from omicron. I am just leaving it here for now meanwhile we have to use the operation ID workaround https://github.com/oxidecomputer/omicron/pull/1329#issuecomment-1175806211

**Comment by david-crespo** [comment]
Why do you need to check in spec.json for the workaround? Just to have something to reference to understand where the transformed names are coming from?

**Comment by karencfv** [comment]
Yes, and also to copy it to the CLI

**Pull request merged**